services/ui-src/src/utils/forms/forms.ts

```diff
@@ -101,8 +102,11 @@ export const initializeDropdownFields = (fields: FormField[]) => {
     (field: FormField) => field.type === "dropdown"
   );
   dropdownFields.forEach((field: FormField) => {
-    // if first option is not already a blank default value
-    if (field?.props?.options[0].value !== "") {
+    // if first provided option is not already a blank default value
+    if (
+      typeof field?.props?.options !== "string" &&
+      field?.props?.options[0].value !== ""
+    ) {
```

Comment:
Rather than do this in two different places, we should pull this into the dropdown field component instead. Makes it less likely that we'll introduce some bug if we change it in the future.